main.c

```diff
@@ -0,0 +1,21 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+int* max(int *first_num, int *second_num)
+{
+	(*first_num > *second_num) ? (return first_num) : (return second_num);
+}
+
+int main(void)
+{
+	int first_number     = 0;
+	int second_number    = 0;
+	int *first_number_P  = &first_number;
+	int *second_number_P = &second_number;
+
+	printf("first_number \n");
+	scanf("%d", first_number_P);
+	printf("second_number \n");
+	scanf("%d", second_number_P);

```

Comment:
😎Question
Try to assign NULL and see what will happen?
What you missed in max(int *first_num, int *second_num) implementation?

Add line:
int *first_number_P  = NULL;

Reply:
Done